### PR TITLE
Fix for #114

### PR DESCRIPTION
**Pull request by nickjbrowning**

### CODE REVIEW

**Review by Luthaf — APPROVED** [review]

**Comment by Luthaf** [comment]
The code looks good, we can merge once CI is happy

File: sphericart-torch/include/sphericart/torch.hpp [hunk 1]
```diff
@@ -45,8 +46,8 @@ class SphericalHarmonics : public torch::CustomClassHolder {
     sphericart::SphericalHarmonics<float> calculator_float_;
 
     // CUDA implementation
-    sphericart::cuda::SphericalHarmonics<double> calculator_cuda_double_;
-    sphericart::cuda::SphericalHarmonics<float> calculator_cuda_float_;
+    sphericart::cuda::SphericalHarmonics<double>* calculator_cuda_double_ptr = nullptr;
```
Comment (Luthaf) [inline sphericart-torch/include/sphericart/torch.hpp:49]: please use modern C++ here =) This should be `std::unique_ptr`!

**Review by Luthaf — CHANGES_REQUESTED** [review]

File: sphericart-torch/include/sphericart/torch.hpp (hunk 1 above)
Comment (nickjbrowning) [reply sphericart-torch/include/sphericart/torch.hpp]: fixed in latest commit.

**Comment by frostedoyster** [comment]
@nickjbrowning @Luthaf what's the status of this?

**Comment by Luthaf** [comment]
This was waiting on CI to be fixed, after what I can do a quick review and merge!

**Comment by nickjbrowning** [comment]
should be good for review @Luthaf

**Review by Luthaf — APPROVED** [review]

**Pull request merged**